packages/linux/package.mk

```diff
@@ -29,6 +29,32 @@ PKG_SHORTDESC="linux26: The Linux kernel 2.6 precompiled kernel binary image and
 PKG_LONGDESC="This package contains a precompiled kernel image and the modules."
 PKG_IS_KERNEL_PKG="yes"
 
+PKG_CFG_FILE="$PKG_NAME.${TARGET_PATCH_ARCH:-$TARGET_ARCH}.conf"
```

Comment:
this block can be removed/simplified after #2432 no?

Reply:
No, this boiler plate hasn't been simplified (yet) - we could add a helper function `get_kernel_config_file()` that will simplify/centralise this. I'll look at it for a separate PR.

Reply:
Added #2485